test/e2e/specs/editor/various/footnotes.spec.js

```diff
@@ -60,6 +60,13 @@ test.describe( 'Footnotes', () => {
 			},
 		] );
 
+		// Check if the numbers in the editor content updated.
+		const anchorNumber = await editor.canvas
+			.locator( ':root' )
+			.evaluate( () => document.querySelector( '.fn' ).textContent );
+
+		expect( anchorNumber ).toBe( '1' );
```

Comment:
Can we use `await expect( editor.canvas.locator( '.fn' ) ).toHaveText( '1' )` here?

Reply:
Fancy. It's not clear to me what this does. Does it make sure there's a match for "1" or does it make sure it is exactly "1"? https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text

I've tripped over these Playwright locators before so I like to stick to plain old DOM APIs.

Reply:
Do you mean `contains` vs `equals`?

* `toHaveText( '1' )` - `node.textContent === '1'`
* `toContainText( '1' )` - `node.textContent.includes( '1' )`

In my experience, e2e tests rarely end up being flaky when sticking with Playwright APIs.

Reply:
Ok, I didn't say equals anywhere in the docs